bql/semantic/expression_test.go

```diff
@@ -715,7 +824,7 @@ func TestEvaluatorEvaluateError(t *testing.T) {
 
 		got, err := eval.Evaluate(entry.r)
 		if err == nil {
-			t.Errorf("%s: eval.Evaluate(%v) = _, nil; want non-nil error", entry.id, entry.r)
+			t.Errorf("%s: eval.Evaluate(%v) = _, nil; want _, non-nil error", entry.id, entry.r)
 		}
 		if want := entry.want; got != want {
```

Comment:
`want` is always false for `TestEvaluatorEvaluateError` test cases right? You could remove it from the test struct and simply compare got with `false`.

```
if want := false; got != want { ... }
```

Reply:
Thank you once again for pointing this Tati, I have just done so. 🙂